src/main/java/org/datadog/jenkins/plugins/datadog/DatadogGlobalConfiguration.java

```diff
@@ -1045,7 +1045,7 @@ public void setCollectBuildTraces(boolean collectBuildTraces) {
     /**
      * @return - A {@link Boolean} indicating if the user has configured Datadog to enable CI Visibility.
      */
-    public boolean isEnabledCiVisibility() {
+    public boolean isEnableCiVisibility() {
```

Comment:
here is the problem:
wrong: `isEnabledCiVisibility`
correct: `isEnableCiVisibility`